app/controllers/application_controller.rb

```diff
@@ -1,7 +1,6 @@
 class ApplicationController < ActionController::Base
   before_action :exclude_all_users_except_admins_during_maintenance
   before_action :authenticate
-  before_action :set_paper_trail_whodunnit
```

Comment:
Should this have gone away? `paper_trail` is still in use, I think?

Reply:
I can't actually find that method, and I can't find it in the alphagov transition version either so I'm not sure what it refers to 🤔

Reply:
Oh wait it's a papertrail method included in the gem!